### PR TITLE
Add styles to fintech guide

### PR DESCRIPTION
**Pull request by rbruggem**
Also, add guide and sections to sidebar.

This PR is based on #2 which won't be merged.

### CODE REVIEW

File: docs/guides/integrate-fintech.md [hunk 1]
```diff
@@ -0,0 +1,450 @@
...
+to integrate CO₂ emissions estimations and offsetting into an existing end-user experience.
+
+Feel free to contact our [support team](mailto:support@lune.com) if you encounter any issues integrating our logistics use case.
+
+</div>
+<div>
+
+## Concepts
+
+- **Client account** - You must create a client account for each of your clients to define their basic characteristics, such as name, currency, and beneficiary.  Clients will be provided a Lune defaults
```
Comment (raguiar9080) [inline docs/guides/integrate-fintech.md:48]: Does it make sense to have some formatting on special formatting on `Lune defaults` here to symbolize it's a concept we have? Something like what we do with Project bundles examples? _Lune defaults_?

File: docs/guides/integrate-fintech.md [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+### Sample request
+
+**Where**:
+
+- `$CLIENT_ACCOUNT_ID` is the unique identifier for the client
+- `value.value` defines the shipment load; in this example, 40 tonnes
+- `value.currency` is the currency in which the transaction was completed
+- `merchant.category_code` is the Merchant Category Code (MCC) used to classify the type of goods and services that are transacted
+- `merchant.name` is the name of merchant in the transaction
+- `merchant.country_code` is the 3-character (which standard? ISO) country code for the country in which the transaction was completed
```
Comment (raguiar9080) [inline docs/guides/integrate-fintech.md:113]: Is the `which standard?` question supposed to be here 😄

File: docs/guides/integrate-fintech.md [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+<div>
+
+### Sample response
+
+A successful 200 request will return the total cost of offsetting the CO₂ emissions, along with options for offsetting (Lune defaults) which can be presented in-app:
+
+**Where**:
+
+- `estimated_total_cost` is the total cost of offsetting the CO₂ emissions in the client's currency
+- `bundles` is a container object for the Project bundles that will be used to offset the emissions associated with the transactions and includes the bundle id, bundle name, and unit price
```
Comment (raguiar9080) [inline docs/guides/integrate-fintech.md:282]: not sure what writing style we usually aim for, but I think when we use `and` at the end we don't want a comma immediately before.

File: docs/guides/integrate-fintech.md [hunk 4]
```diff
@@ -0,0 +1,450 @@
...
+</div>
+<div>
+
+### Sample response
+
+**Where**:
+
+- `id` is the unique identifier for the booking
+- `idempotency_key` is a unique token that you submit as a request header, that guarantee that only one order will be created regardless of how many times a request is sent to us
+- `status`is the order status.  `placed` means the order has been validated and accepted and will be fulfilled
```
Comment (raguiar9080) [inline docs/guides/integrate-fintech.md:373]: double space before `placed`

Also, double `and` is a bit weird, I would do: `placed` means the order has been validated, accepted and will be fulfilled

File: docs/guides/integrate-fintech.md [hunk 5]
```diff
@@ -0,0 +1,450 @@
...
+
+</div>
+<div>
+
+### Sample response
+
+**Where**:
+
+- `id` is the unique identifier for the booking
+- `idempotency_key` is a unique token that you submit as a request header, that guarantee that only one order will be created regardless of how many times a request is sent to us
```
Comment (raguiar9080) [inline docs/guides/integrate-fintech.md:372]: Typo, should be `that guarantees`

**Review by raguiar9080 — APPROVED** [review]
Left some small comments on typos and improvements but leaving approval, looking nice 👍

File: docs/guides/integrate-fintech.md (hunk 2 above)
Comment (rbruggem) [reply docs/guides/integrate-fintech.md:113]: oops

File: docs/guides/integrate-fintech.md (hunk 3 above)
Comment (rbruggem) [reply docs/guides/integrate-fintech.md:282]: I thought so too.
Both James and Grammarly add a comma.

**Pull request merged**